src/com/rakovets/course/challenge/Enemy.java

```diff
@@ -0,0 +1,41 @@
+package com.rakovets.course.challenge;
+
+public class Enemy implements Mortal {
+
+    int health;
+    String name;
+
+    Enemy(int health, String name) {
+        this.health = health;
+        this.name = name;
+    }
+
+    public int getHealth() {
+        return health;
+    }
+
+    public void setHealth(int health) {
+        this.health = health;
+    }
+
+    public void takeDamage(int damage) {
+        this.health -= damage;
+    }
+
+    public boolean isAlive() {
+        if (this.health > 0) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    public void attackHero(Hero hero) {
+        System.out.println("*Враг атакует героя*");
+        hero.health -= 10;
+        if (hero.health <= 0) {
+            System.out.println(hero.getName() + " died");
+        }
```

Comment:
Very bad! Because Enemy control health hero -> enemy can set any health for the hero